src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -716,7 +716,7 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, workspaceDAO:
     if (acls.maximumAccessLevel >= requiredLevel)
       codeBlock
     else
-      RequestComplete(StatusCodes.Forbidden, "User %s does not have the required access to the requested resource")
+      RequestComplete(http.StatusCodes.NotFound, s"${workspace.namespace}/${workspace.name} does not exist")
```

Comment:
What is the message returned when the workspace truly does not exist?  To ensure they're identical, it would be nice for both of these to reference a common (parameterized) string value.